pkg/datapath/linux/routing/routing.go

```diff
@@ -58,7 +59,8 @@ func (info *RoutingInfo) Configure(ip net.IP, mtu int, masq bool) error {
 		Mask: net.CIDRMask(32, 32),
 	}
 
-	// Route all traffic to the ENI address via the main routing table
+	// Route all traffic to the ENI address via the main routing table. Egress
+	// rules are an exception.
```

Comment:
I still don't get "all traffic *to* the ENI address and then "Egress rules are an exception", the egress rules don't have the ENI address as a destination right?

Reply:
Reword. I think what it was intending to say is that the traffic is routed to the EP IP rather than the "ENI address". The "egress rules are an exception" part is referring to the ingress rules associated with the main routing table, which is not the case for egress rules, hence the comment. I also clarified this part.